### DISCUSSION TITLE
Question: Way to fetch boosts/reblogs from public timeline via API?

### DISCUSSION THREAD

**Discussion by ummjackson** [mastodon/mastodon#21776] [Q&A]
Hi there, I've been scratching my head on this one after realizing that by default, the public timeline API method `api/v1/timelines/public` doesn't return reblogs (aka boosts). Is there any way to retrieve these via an API?

**Comment by amosbatista** [comment]
Reactivating this thread.
I've looked into other issues and Q&A pages, but I didn't find a way to show boosts in `\timelines` entrypoint.
There's no option to show it in timeline entrypoint.

Is there really no way to show the boost in `\timelines`?

**Reply by ummjackson** [reply]
Not in public timelines. You'll get boosts in the home timeline from people you follow, but not in the public federated or local timelines. Would require code changes from the Mastodon side.
